poker-pals/src/App.vue

```diff
@@ -11,6 +11,8 @@
 // import Profile from './components/Profile.vue';
 // import Reports from './components/Reports.vue';
 // import Tables from './components/Tables.vue';
+//import Table from './components/Table.vue';
```

Comment:
I created `Tables` component in order to host the changes you made. If you are already using your own component, we can probably delete the one i made. You could also shift the logic into `Tables` component. Up to you 😃